src/Prefixer.js

```diff
@@ -59,6 +64,17 @@ export function prefix (value, length) {
 		// justify-content
 		case 4968:
 			return replace(replace(value, /(.+:)(flex-)?(.*)/, WEBKIT + 'box-pack:$3' + MS + 'flex-pack:$3'), /s.+-b[^;]+/, 'justify') + WEBKIT + value + value
+		// justify-self
+		case 4200:
+			// center, end, start, stretch
+			if (
+				~indexof(['stretch', 'center', 'end', 'start'], substr(value, length + 1, sizeof(value) - 1))
```

Comment:
TODO: convert to regex, not Array.indexOf.

Reply:
Fixed in https://github.com/thysultan/stylis/pull/285/commits/817ff44b0bbc87f950e25b9e35f72d7531f2ee81